scripts/make_video.py

```diff
@@ -0,0 +1,154 @@
+import argparse
+import random
+
+import torch
+import yaml
+
+from diffusers import DPMSolverMultistepScheduler
+from stable_diffusion_videos import StableDiffusionWalkPipeline
+
+
+def init_arg_parser():
```

Comment:
Since we're already installing `fire` with the requirements of the package, maybe lets just use that instead? I can update to do this so its not a hassle for you :)

Reply:
I am not too familiar with `fire` but I can give it a try. Tho after quickly skimming the docs, while this would considerably reduce boilerplate, I think I prefer the flexibility of `argparse`. E.g. I prefer calling
```shell
python scripts/make_video.py --prompts "a cat" "a dog" --seeds 42 1337
```
over
```shell
python scripts/make_video.py --prompts="['a cat', 'a dog']" --seeds=[42,1337]  # note that --seeds=[42, 1337] would fail!
```
Moreover I can feel some dirty hacking would be required to keep support for argument provision through config file using the `--cfg` option, which is an important feature IMO.

Let me know what you think. If this is something you really require then I will give it a shot.

Reply:
Interesting. I think I agree with you! will have a look when I can